yarn.lock

```diff
@@ -14385,7 +14385,7 @@ pretty-format@^26.0.0, pretty-format@^26.4.0, pretty-format@^26.5.2, pretty-form
     ansi-styles "^4.0.0"
     react-is "^17.0.1"
 
-pretty-format@^27.0.6, pretty-format@^27.2.0:
+pretty-format@^27.2.0:
```

Comment:
not sure why this changed, revert

Reply:
With `yarn-lock-deduplicate` integrated at root level, when someone only runs `yarn` in package or module directory the deduplication script do not run. If next person then runs `yarn` on the root level, the lock is being updated:
* https://github.com/expo/expo/commit/9a921252f1b7ff5303a9d2be562568ea550700b6